src/SignalR/clients/ts/signalr/src/ServerSentEventsTransport.ts

```diff
@@ -56,9 +58,10 @@ export class ServerSentEventsTransport implements ITransport {
                 return;
             }
 
+            const withCredentials = this.withCredentials === undefined ? true : this.withCredentials;
```

Comment:
Do we need all these `withCredentials === undefined` checks? If so, why isn't the check in the HttpConnection ctor sufficient by itself?

Reply:
I can remove this specific one, but not the XhrHttpClient one unless we want to make HttpRequest have `boolean` instead of `boolean | undefined`. But we usually let public types have properties that can be undefined. And we can't provide a default value for interfaces.

Reply:
I see. That seem fair.